versions/1.0.0.md

```diff
@@ -238,14 +238,14 @@ version: 1.0.1
 
 Describes a source description (such as an OpenAPI description) that will be referenced by one or more workflows described within a Workflows Description.
 
-An object storing a map between named description keys and location URLs to the source descriptions (such as an OpenAPI description) this workflow SHALL apply to. Each source location `string` MUST be in the form of a [URL](https://url.spec.whatwg.org/). MAY be defined using relative references as defined by [RFC3986](https://spec.openapis.org/oas/latest.html#bib-RFC3986).
+An object storing a map between named description keys and location URLs to the source descriptions (such as an OpenAPI description) this workflow SHALL apply to. Each source location `string` MUST be in the form of a URL and MAY be defined using relative references as defined by [RFC3986](https://spec.openapis.org/oas/latest.html#bib-RFC3986).

```

Comment:
Thanks for settling on RFC 3986!  I'd be fine with this, but you could change it to:

> MUST be in the form of a URI-reference as defined by RFC3986 section 4.1.

You have the "location URL" phrase which is the correct term for the usage, and "URI-reference" is the correct term for the syntax.